src/test/scala/com/microsoft/hyperspace/index/IndexLogEntryTest.scala

```diff
@@ -362,33 +362,10 @@ class IndexLogEntryTest extends HyperspaceSuite with SQLHelper {
     assert(directoryEquals(actual, expected))
   }
 
-  test("Directory.fromLeafFiles: throwIfNotExist flag throws exception for non-existent" +
```

Comment:
Don't we still need a test case with nonexistent directory?

Reply:
I think below `Directory.fromDirectory where the directory is empty.` is enough